src/content/general/releases/index.md

```diff
@@ -96,7 +97,7 @@ Once deprecated, you can still continue to use the workload cluster release with
 
 You have several options to inspect workload cluster release details:
 
-- All workload cluster releases are defined in code in the [giantswarm/releases](https://github.com/giantswarm/releases) GitHub repository. Here you also find comprehensive release notes.
+- We announce new workload cluster releases in your Slack support channel. There you will find a link to the according release notes in our [changes and releases]({{< relref "/changes" >}}) section here in the docs site. Here you also find comprehensive release notes.
```

Comment:
```suggestion
- We announce new workload cluster releases in your Slack support channel. In each announcement, you will find a link to the corresponding release notes in our [changes and releases]({{< relref "/changes" >}}) section here on the docs site, where you also find comprehensive release notes.
```

Reply:
Thanks! Always glad for language corrections by those who should know better :)